### PR TITLE
✏️ Adds definition for 'mx'

### PR DESCRIPTION
**Pull request by tatianamac**

### CODE REVIEW

File: 11ty/definitions/mx.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Mx.
+slug: mx
+speech: noun
+defined: true
+excerpt: someone who is sexually attracted to people regardless of their gender, or to people of any gender.
```
Comment (ZoeBijl) [inline 11ty/definitions/mx.md]: Was this copied from somewhere, doesn't look quite right?

File: 11ty/definitions/mx.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  - text: 'This Gender-Neutral Word Could Replace Mr. and Ms.'
+  - text: 'Merriam-Webster definition'
+    href: https://www.merriam-webster.com/dictionary/MX
+---
+
+gender-neutral honorific or title of courtesy, equivalent to Ma'am, Madam, Miss, Mr, Mrs, Mrs, Sir.
+
+## Considerations
+
+In formal settings, defaulting to gendered titles of courtesy can misgender people. While titles and salutations are not necessarily indicative of gender, there tends to be a corrolation between **Ma'am** and woman- or femme-assumed individuals.
```
Comment (ZoeBijl) [inline 11ty/definitions/mx.md]: Well written ✨

**Review by ZoeBijl — CHANGES_REQUESTED** [review]
I don't think the excerpt is quite right.

File: 11ty/definitions/mx.md (hunk 1 above)
Comment (tatianamac) [reply 11ty/definitions/mx.md]: Ooh, good catch. thank you!

**Comment by tatianamac** [comment]
Thank you for reviewing, @ZoeBijl . Would you mind taking a look at my revisions please? 🙏🏽

**Pull request merged**